core/src/test/scala/integration/kafka/api/BaseQuotaTest.scala

```diff
@@ -335,12 +339,31 @@ abstract class QuotaTestClients(topic: String,
       assertEquals("Should not have been throttled", 0.0, metricValue(maxMetric), 0.0)
   }
 
-  def quotaProperties(producerQuota: Long, consumerQuota: Long, requestQuota: Double): Properties = {
-    val props = new Properties()
-    props.put(DynamicConfig.Client.ProducerByteRateOverrideProp, producerQuota.toString)
-    props.put(DynamicConfig.Client.ConsumerByteRateOverrideProp, consumerQuota.toString)
-    props.put(DynamicConfig.Client.RequestPercentageOverrideProp, requestQuota.toString)
-    props
+  // Default is set by using Some(null) for either or both user and clientId
```

Comment:
Maybe add a constant for the default entity and use `Some(DefaultEntity)` which would avoid the need for the comment that sounds like we are doing something we shouldn't be.